mm.lic

```diff
@@ -0,0 +1,94 @@
+=begin
+  Documentation: https://elanthipedia.play.net/Lich_script_repository#mm
+=end
+custom_require.call(%w[common-arcana events])
+
+class MoonMagery
+  include DRCA
+  def initialize
+    args = get_args
+    @settings = get_settings
+
+    if args.locate
+      locate?(args.person)
+    elsif args.rift
+      rift(args.person)
+    end
+    Flags.delete('locate-fail')
```

Comment:
Should this deletion be done in `locate?`? I.e. it feels like delete should be in the same 'scope' as the add. Watch out for early returns though.

Reply:
The early returns are why i moved it out of scope. I gave preference to only doing the delete on one line. In keeping with that preference, I could make `create_flags` `destroy_flags` methods which takes care of all the scripts flags.

Reply:
That makes sense, I'm fine with whatever you think is best.